__tests__/app.test.js

```diff
@@ -9,21 +9,72 @@ beforeEach(() => seed(testData));
 afterAll(() => db.end());
 
 describe("1 GET categories /api/categories", () => {
-    it("status 200, should return the first item from an array of categories", () => {
-      return request(app)
-        .get("/api/categories")
-        .expect(200)
-        .then(({ body }) => {
-            expect(
-                body.forEach((treasure) => {
-                  expect(treasure).toEqual(
-                    expect.objectContaining({
-                        description: expect.any(String),
-                        slug: expect.any(String),
-                    })
-                  );
-                })
-              );
-            });
+  it("status 200, should confirm a return of categories containing a description and a slug", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(
+          body.forEach((category) => {
```

Comment:
when testing arrays, ensure to have an array length expect statement too, as forEach can otherwise easily create false positives